test-requirements.txt

```diff
@@ -1,6 +1,9 @@
+##Install the required packages using pip sudo pip install -r test-requirements.txt
```

Comment:
split the message and command in two lines, Ex:
##Install the required packages using pip sudo
## sudo pip install -r test-requirements.txt